deploy/chart/templates/0000_50_olm_07-olm-operator.deployment.yaml

```diff
@@ -17,6 +17,13 @@ spec:
       labels:
         app: olm-operator
     spec:
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+        {{- if eq .Values.installType "upstream" }}
+        runAsUser: {{ .Values.package.securityContext.runAsUser }}
+        {{- end }}
```

Comment:
```suggestion
```
We cannot use RunAsUser at all,
That is ONLY required because the Dockerfile image is not defining the user when should be.
So, what is the image used here?

Reply:
That's a fair point actually. Since we are baking the USER into the olm container we don't need runAsUser for the controller deployments. Only for the catalog source and unpack jobs, since we have less control there.